presidio-structured/tests/test_tabular_engine.py

```diff
@@ -0,0 +1,74 @@
+from unittest.mock import Mock
+import pandas as pd
+
+import pytest
+
+from presidio_anonymizer.entities import OperatorConfig
+
+from presidio_structured import StructuredEngine
+from presidio_structured.data.data_processors import JsonDataProcessor
+
```

Comment:
in another iteration, I would add more assertions, on the explanation, scoring, etc

Reply:
Good idea :)